festipals/src/components/ProfilePage.js

```diff
@@ -57,7 +77,7 @@ export default class ProfilePage extends Component {
           <hr />
           <div className="row">
             <div className="col-xs-12">
-              <h2>Pending requests <span className="badge">{pendingPalRequests.length + pendingActInvites.length}</span></h2>
+              <h2>{this.state.data.pendingPalRequests.toString} Pending requests <span className="badge">{pendingPalRequests.length + pendingActInvites.length}</span></h2>
```

Comment:
I was just trying to see if I could see a string resp. of the array, but it didnt work, We need to figure out how to access arrays in react or if it is a problem with the mongoose model. :)